README.md

```diff
@@ -1,288 +1,175 @@
--->
-
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
-
-# 📗 Table of Contents
-
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🛠 Built With
 
-> Describe between 1-3 key features of the application.
+### 🧰 Tech Stack
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB)
+- ![Redux](https://img.shields.io/badge/redux-%23593d88.svg?style=for-the-badge&logo=redux&logoColor=white)
+- ![React Router](https://img.shields.io/badge/React_Router-CA4245?style=for-the-badge&logo=react-router&logoColor=white)
+- ![NodeJS](https://img.shields.io/badge/node.js-6DA55F?style=for-the-badge&logo=node.js&logoColor=white)
+- ![Testing-Library](https://img.shields.io/badge/-TestingLibrary-%23E33332?style=for-the-badge&logo=testing-library&logoColor=white)
+- ![Render](https://img.shields.io/badge/Render-%46E3B7.svg?style=for-the-badge&logo=render&logoColor=white)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
+### 🔥 Key Features
+### 🔥 Key Features
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
```

Comment:
- [ ] I know you might've forgotten, but it's ok. This only means you're 5 mins away from the approval 😉 Please record a 5 mins short presentation video to show off what you've done so far on this great capstone 💪

_Do not forget to attach the link proudly in the README file ❤️_

Reply:
Hi @MahmoudBakr23 ,
Thank you for your reviews. However, the code extract you pointed out is an old code. It has been updated in the most recent PR.  The live demo link in the recent PR is in Line 56.
Please do kindly let me know If I am wrong in any way.
Thank you

Reply:
Hi, @Mekuseo Thank you for communicating.

I can see the confusion here. However, I still don't see any link to the [video presentation](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20record%20a%20video%20presenting%20the%20features%20of%20the%20project%20you%20built.%20It%20is%20a%20video%20with%20a%20maximum%20length%20of%205%20minutes.) whether in the PR summary or the README file. On the other side of the confusion, I know that your app is deployed, accessed, and well-working and I highlighted this point in my review ⬆️ 😄

I hope this clears things out, please reach out again for further questions. 🙏

Reply:
Thank you for the clarifications. A link to the video presentation has been added.